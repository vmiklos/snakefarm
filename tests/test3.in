debug 1
out 1
time 100
load test3.map
addplayer 1
addsnake 1 1 0 1 1 0
25 0
24 0
23 0
endsnake
showsnake 1
step
showsnake 1
turnleft 1
step
showsnake 1
step
showsnake 1
turnright 1
step
showsnake 1
exit