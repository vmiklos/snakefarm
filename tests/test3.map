1;W;2;8
2;W;3;9
3;W;4;10
4;W;5;11
5;W;6;12
6;W;7;13
7;W;0;14

8;W;9;15
9;0;10;16
10;0;11;17
11;0;12;18
12;0;13;19
13;0;14;20
14;W;0;21

15;W;16;22
16;0;17;23
17;0;18;24
18;0;19;25
19;0;20;26
20;0;21;27
21;W;0;28

22;W;23;29
23;0;24;30
24;0;25;31
25;0;26;32
26;0;27;33
27;0;28;34
28;W;0;35

29;W;30;0
30;W;31;0
31;W;32;0
32;W;33;0
33;W;34;0
34;W;35;0
35;W;0;0