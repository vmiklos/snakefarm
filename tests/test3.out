Event MapLoad test3.map
Snake 1
Prop 0 1 1
1 25 0
2 24 0
3 23 0
endsnake
Event Step 1
End Step
Snake 1
Prop 0 1 1
1 26 0
2 25 0
3 24 0
endsnake
Event Step 1
StepEvent TurnLeft
End Step
Snake 1
Prop 0 1 1
1 19 0
2 26 0
3 25 0
endsnake
Event Step 1
End Step
Snake 1
Prop 0 1 1
1 12 0
2 19 0
3 26 0
endsnake
Event Step 1
StepEvent TurnRight
End Step
Snake 1
Prop 0 1 1
1 13 0
2 12 0
3 19 0
endsnake
Event Exit
